releases/sui/2025-01-ITS-v1.1.0.md

```diff
@@ -0,0 +1,120 @@
+# Sui ITS v1.1.0
+
+|  | **Owner** |
+|-----------|------------|
+| **Created By** | @blockchainguyy <ayush@interoplabs.io> |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io> |
+
+| **Network** | **Deployment Status** | **Date** |
+|-------------|----------------------|----------|
+| **Devnet Amplifier** | - | TBD |
+| **Stagenet** | - | TBD |
+| **Testnet** | - | TBD |
+| **Mainnet** | - | TBD |
+
+[Release](https://github.com/axelarnetwork/axelar-cgp-sui/releases/tag/v1.1.0)
+
+[Changelog](https://github.com/axelarnetwork/interchain-token-service/blob/v/CHANGELOG.md#210)
+
+## Deployment
+
```

Comment:
The gas service contract API was changed as part of this release. So the gas service needs to redeployed (we opted for that instead of upgrade for simplicity, and since it hasn't been used in prod yet)

https://github.com/axelarnetwork/axelar-cgp-sui/commit/2603003c62df9af925a35e00a9c00789c557c45c

Implications:
- Delete existing gas service/example
- Deploy new one
- Transfer cap to Operators contract
- Deploy ITS
- Redeploy Example

Aiming to do the release tomorrow, were blocked on some things